docs: add routes and files (#852)
flybywiresim/docs: docs/fbw-a380x/routes/operating/AAR.md

# Asiana Airlines (AAR/OZ)

---

## ^^Fleet^^

> Asiana Airways has 6 A380s in it's fleet first delivered in December 2013 and last on August 2016

Reviewer: ```suggestion
Asiana Airways has 6 A380 in its fleet first delivered in December 2013 and last on August 2016

```

---

### HL7625
### HL7626
### HL7634
### HL7635
### HL7640
### HL7641

---

## ^^Routes^^

> Asiana is currently operating their A380s on 3 routes around the world.

Reviewer: ```suggestion
Asiana is currently operating their A380 on 3 routes around the world.

```

![AAR Routes](../../../fbw-a38nx/assets/routes/aarroutes.jpg "AAR Routes")

### **Currently operating**

> | Departure ICAO/IATA | Arrival ICAO/IATA | Flight Number | Lenght | Distance |
> |:--------|:--------|:------|:------|:-----|
> | RKSI/ICN | VTBS/BKK | OZ741 | 5h15m | 1980nm |
> | VTBS/BKK | RKSI/ICN | OZ742 | 4h40m | 1980nm |
> | RKSI/ICN | YSSY/SYD | OZ601 | 10h00m | 4550nm |
> | YSSY/SYD | RKSI/ICN | OZ602 | 10h10m | 4550nm |
> | RKSI/ICN | KLAX/LAX | OZ202 | 10h10m | 5210nm |
> | KLAX/LAX | RKSI/ICN | OZ201 | 12h55m | 5210nm |

Reviewer: ```suggestion
| Departure ICAO/IATA | Arrival ICAO/IATA | Flight Number | Length | Distance |
|:--------|:--------|:------|:------|:-----|
| RKSI/ICN | VTBS/BKK | OZ741 | 5 h 15 m   | 1980 NM |
| VTBS/BKK | RKSI/ICN | OZ742 | 4 h 40 m   | 1980 NM |
| RKSI/ICN | YSSY/SYD | OZ601 | 10 h 00 m | 4550 NM |
| YSSY/SYD | RKSI/ICN | OZ602 | 10 h 10 m | 4550 NM |
| RKSI/ICN | KLAX/LAX | OZ202 | 10 h 10 m | 5210 NM |
| KLAX/LAX | RKSI/ICN | OZ201 | 12 h 55 m | 5210 NM |

```
